### PR TITLE
Expanded on CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by pjackson28**
- Added sections regarding general changes, content changes and view changes

### CODE REVIEW

**Comment by pjackson28** [comment]
@bryan-robitaille @nschonni Welcome your feedback on this. Would like for it to make sense for contributors, helping them to avoid problem areas.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,19 +1,87 @@
...
 
 Feel free to propose changes by creating Pull Requests. If you don't have write access, editing a file will create a Fork of this project for you to save your proposed changes to. Submitting a change to a file will write it to a new Branch in your Fork, so you can send a Pull Request.
 
 If this is your first time contributing on GitHub, don't worry! Let us know if you have any questions.
 
+## General changes
+
+All proposed changes will go through two levels of review:
+
+1. Automated checks (e.g., markdownlint, link checks)
```
Comment (nschonni) [inline CONTRIBUTING.md]: Might need some additional guidance here, or maybe look at common rules people will trip on. Also/alternately start adding child `.markdownlint.json` if stuff like the views have/need different rules

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,19 +1,87 @@
...
+1. Manual checks (e.g., manuak content review, code inspection)
```
Comment (nschonni) [inline CONTRIBUTING.md]: Typo: `manuak`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,19 +1,87 @@
...
+1. Don't change or remove the <code>div</code> and <code>section</code> elements that have <code>class="dpgn-section-&lt;section-name&gt;"</code>. These are used as landmarks for dataset generation.
+1. Don't change the standards names or the names of the first four guidelines for each standard (e.g., 1.1, 1.2, 1.3 and 1.4). These names come directly from the Digital Standards so should be changed there first before they can be changed in the Digital Playbook.
+1. The names for "Guidelines", "Related guidelines", "Checklist", "Implementation guides", "Reusable solutions" and "Similar resources" (as well as the French versions) are centrally managed. Changing them in an authoring <code>.md</code> file will have no effect on the rendered result.
+1. Content that shouldn't be included in the dataset should be excluded by applying the <code>dpgn-data-ignore</code> class. This can be useful for breaking up the authoring <code>.md</code> file to make it more legible without affecting the dataset. This can also be useful for creating content groups for applying common tags (e.g., applying a tag to a whole list rather than each individual list item).
```
Comment (nschonni) [inline CONTRIBUTING.md]: An example of using the `dpgn-data-ignore` might be helpful, even if just pointing to a hash/line in an existing file

**Comment by pjackson28** [comment]
I addressed all the issues that were raised. Thank you for the feedback!

**Pull request merged**